sql/mozfun/event_analysis/event_property_value_to_match_string/udf.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE FUNCTION event_analysis.event_property_value_to_match_string(
+  property_index INTEGER,
+  property_value STRING
+)
+RETURNS STRING AS (
+  CONCAT(
+    event_analysis.escape_metachars(property_value),
+    -- Event property values are stored in `events_daily.events` strings in reverse order,
+    -- so we expect the Nth property value to be followed by N-1 other property values.
+    IF(property_index > 1, CONCAT('[^,]{', (property_index - 1), '}'), '')
```

Comment:
This is the crux of the fix for mozilla/lookml-generator#281, as the [existing logic](https://github.com/mozilla/lookml-generator/blob/10d8577c55e4cf607c43811ee4d5d214a66a0d26/generator/views/funnel_analysis_view.py#L166-L175) only matches the first property character immediately preceding the event character.